### PR TITLE
fix(s.weibo.com): fix navigator and logo color for dynamic mode

### PR DESCRIPTION
**Pull request by Asukaaaaaa**
before:
image

after:
image

### CODE REVIEW

File: src/config/dynamic-theme-fixes.config [hunk 1]
```diff
@@ -19441,6 +19441,19 @@ INVERT
 
 ================================
 
+s.weibo.com
+
+CSS
+[data-darkreader-inline-bgcolor] {
```
Comment (alexanderby) [inline src/config/dynamic-theme-fixes.config:19447]: Hi, could you please try targeting those exact nodes by classes or other attributes? For example something like:
```
.class {
    background-color: var(--darkreader-neutral-background) !important;
    border-color: ${#cccccc} !important;
}
```

**Review by alexanderby — CHANGES_REQUESTED** [review]

File: src/config/dynamic-theme-fixes.config (hunk 1 above)
Comment (Asukaaaaaa) [reply src/config/dynamic-theme-fixes.config]: done🫡

File: src/config/dynamic-theme-fixes.config [hunk 2]
```diff
@@ -19441,6 +19441,19 @@ INVERT
 
 ================================
 
+s.weibo.com
+
+INVERT
+#woo_svg_nav_logo > g > path:nth-child(5)
```
Comment (alexanderby) [inline src/config/dynamic-theme-fixes.config:19447]: Something feels wrong about this selector, it target's a 5th path element in a logo. BUt I hope you know what you are doing.

File: src/config/dynamic-theme-fixes.config [hunk 3]
```diff
@@ -19441,6 +19441,19 @@ INVERT
...
+
+CSS
+#searchapps > div > div[role=navigation] > div.woo-panel-main {
+    --darkreader-bg--weibo-top-nav-panel-bgColor: rgb(24, 26, 27);
```
Comment (alexanderby) [inline src/config/dynamic-theme-fixes.config:19451]: Probably CSS variables detection failed, but lets see if your fix works

**Review by alexanderby — APPROVED** [review]

File: src/config/dynamic-theme-fixes.config (hunk 2 above)
Comment (Asukaaaaaa) [reply src/config/dynamic-theme-fixes.config:19447]: image
original fill is covered by a light color, it makes the logo center invisible, so i INVERT that part.😋
image

**Pull request merged**